src/search/FindInFilesUI.js

```diff
@@ -74,16 +74,21 @@ define(function (require, exports, module) {
      * @return {$.Promise} A promise that's resolved with the search results or rejected when the find competes.
      */
     function searchAndShowResults(queryInfo, scope, filter, replaceText, candidateFilesPromise) {
-        return FindInFiles.doSearchInScope(queryInfo, scope, filter, replaceText, candidateFilesPromise)
+        var progressFirstPage, progressFirstUpdate,
+            progressCount = 0,
+            searchAndShowDeferred = new $.Deferred(),
+            searchInScopePromise = FindInFiles.doSearchInScope(queryInfo, scope, filter, replaceText, searchAndShowDeferred.promise());
+        
+        searchInScopePromise
             .done(function (zeroFilesToken) {
                 // Done searching all files: show results
                 if (FindInFiles.searchModel.hasResults()) {
-                    _resultsView.open();
-
-                    if (_findBar) {
-                        _findBar.close();
+                    if (!progressFirstUpdate) {
```

Comment:
I removed this since I thought that we could never get to this point. But I think that it would be good to re-add it: (replace this line with):
```
if (progressFirstUpdate) {
    _resultsView._updateResults();
} else {
```
The alternative would be to check in which page we are, and depending on the result update just the summary or the whole panel.